### PR TITLE
[diaspora] Cleanups and type-hinting

### PR DESCRIPTION
**Pull request by Quix0r**
This cleans up the following:
- added curly braces/spaces for better readability
- don't use count() when you can use `DBA::isResult()`
- used proper type-hints `App` and `array`
- "imported" Friendica\App

### CODE REVIEW

File: diaspora/diaspora.php [hunk 1]
```diff
@@ -9,65 +9,78 @@
...
-function diaspora_queue_hook(&$a,&$b) {
+function diaspora_queue_hook(App $a, &$b) {
 	$hostname = $a->get_hostname();
 
 	$qi = q("SELECT * FROM `queue` WHERE `network` = '%s'",
 		DBA::escape(NETWORK_DIASPORA2)
 	);
-	if(! count($qi))
+
+	if (!DBM::is_result($qi)) {
```
Comment (MrPetovan) [inline diaspora/diaspora.php]: It's `DBA::isResult` now.

File: diaspora/diaspora.php [hunk 2]
```diff
@@ -9,65 +9,78 @@
...
 
 		logger('diaspora_queue: run');
 
 		$r = q("SELECT `user`.* FROM `user` LEFT JOIN `contact` on `contact`.`uid` = `user`.`uid`
 			WHERE `contact`.`self` = 1 AND `contact`.`id` = %d LIMIT 1",
 			intval($x['cid'])
 		);
-		if(! count($r))
+
+		if (!DBM::is_result($r)) {
```
Comment (MrPetovan) [inline diaspora/diaspora.php]: It's `DBA::isResult` now.

**Comment by Quix0r** [comment]
Done and rebased it again new `develop` branch.

**Comment by Quix0r** [comment]
500 error!

**Pull request merged**